### DISCUSSION TITLE
Changing the Default Grid Step to 1 in Excalidraw for Browser Integration

### DISCUSSION THREAD

**Discussion by LDSmj** [excalidraw/excalidraw#8553] [Q&A]
I'm integrating Excalidraw into a browser-based application and want to change the default grid step to 1. 

Here's my current configuration:

```
const options = {
    langCode: 'ja-JP',
    name: name,
    initialData: {
        libraryItems: library.libraryItems,
        elements: [imageElement, ...elements],
        files: files,
        appState: {
            ...scene.appState,
            ...appStateDefault(),
            currentItemFontFamily: 2,
            gridSize: 20, // Controls the grid box size
        },
    },
};
```

I've tried adding 'gridStep: 1' to the appState to change it to a 1x1 grid, but the gridstep still defaults to a 5x5.

How can I correctly change the default grid step to 1x1 instead of the default 5x5? Which part of the code should I update to ensure this works?

**Comment by Mrazator** [comment]
It might be either a bug or something that wasn't meant to be configurable. Will convert to issue.
